README.md

```diff
@@ -1,11 +1,11 @@
 # indieopensource.com Tiny Contributor License Agreement
 
-an tiny contributor license agreement for public-private licensing projects
+A tiny contributor license agreement for public-private licensing projects.
```

Comment:
Hmm. Not a complete sentence.

Reply:
Might I ask you to revert this change?

Reply:
The capitalization and punctuation were more reflexive side effects of editing the "an tiny" to "a tiny".

I was going to ask about this lead, anyways. I'm not sure what public-private licensing means and I think maybe a link would be in order.

Also should we just make the lead/lede complete? "Tiny CLA is a small but sufficient contributor license agreement for public-private licensing projects." Or something like that?

Reply:
Complete sentence? Go for it!